In [12]:
import pandas as pd
import numpy as np
pd.set_option('display.max_columns', None)

In [13]:
df = pd.read_csv('PE_Hospital_Tracker.csv')

In [25]:

grouped = df.groupby('Type').count()
grouped


Name  Facility ID (CCN)  \
Type                                                          
Acute Care - Department of Defense    35                 35   
Acute Care Hospitals                3171               3171   
Childrens                             91                 91   
Critical Access Hospitals           1355               1355   
LTACH                                346                346   
Psychiatric                          612                613   
Rehabilitation hospital              310                310   
Religious non-med                     15                 15   

                                    Parent System/Company  PE Firm  \
Type                                                                 
Acute Care - Department of Defense                      0        0   
Acute Care Hospitals                                  680      192   
Childrens                                              10        0   
Critical Access Hospitals                              62       19   
LTACH                                                 313       78   
Psychiatric                                           298      110   
Rehabilitation hospital                               278       51   
Religious non-med                                       0        0   

                                    Currently PE?  State  Address  City   Zip  \
Type                                                                            
Acute Care - Department of Defense             35     35       35    35    35   
Acute Care Hospitals                         3171   3171     3171  3171  3171   
Childrens                                      91     91       91    91    91   
Critical Access Hospitals                    1355   1355     1355  1355  1355   
LTACH                                         346    346      346   346   346   
Psychiatric                                   613    613      613   613   613   
Rehabilitation hospital                       310    310      310   310   310   
Religious non-med                              15     15       15    15    15   

                                    County  Date last acquired by PE  \
Type                                                                   
Acute Care - Department of Defense      35                         0   
Acute Care Hospitals                  3171                       144   
Childrens                               89                         0   
Critical Access Hospitals             1355                        19   
LTACH                                    0                        76   
Psychiatric                            589                        80   
Rehabilitation hospital                  0                        37   
Religious non-med                        0                         0   

                                    Previous Owner Type  Hospital Ownership  \
Type                                                                          
Acute Care - Department of Defense                    0                  35   
Acute Care Hospitals                                142                3171   
Childrens                                             0                  89   
Critical Access Hospitals                            19                1355   
LTACH                                                75                  87   
Psychiatric                                          79                 601   
Rehabilitation hospital                              36                 169   
Religious non-med                                     0                   0   

                                    Region  Payment Designation  \
Type                                                              
Acute Care - Department of Defense      35                    0   
Acute Care Hospitals                  3171                 3152   
Childrens                               91                    0   
Critical Access Hospitals             1355    

Note: There are 613 facility IDs for psychiatric hospitals but only 612 names. This is because the address provided for that entry leads to Montevista Hospital in Las Vegas, which was shuttered in 2019 by federal regulators after facing hundreds of health and safety complaints. It has reopened as of 2021 as Desert Winds Hospital in the same location.